Arrowgene.Ddon.GameServer/Handler/PartyPartyJoinHandler.cs

```diff
@@ -82,7 +82,15 @@ public override S2CPartyPartyJoinRes Handle(GameClient client, C2SPartyPartyJoin
                         // Skip ourselves
                         continue;
                     }
-                    client.Send(member.GetS2CContextGetParty_ContextNtc());
+
+                    if (member is PlayerPartyMember playerMember)
+                    {
+                        client.Send(playerMember.GetPartyContext());
+                    }
+                    else if (member is PawnPartyMember pawnPartyMember)
+                    {
+                        client.Send(pawnPartyMember.GetPartyContext());
+                    }
```

Comment:
I'm sure this could be an abstract method like before but i struggled with C# type system when doing stuff like this in the past which is why i ended up having them return Packet so it'll do